src/pages/Detail/Components/ReviewModal.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const ReviewModal = ({
+  reviewModal,
+  comment,
+  uploadImg,
+  modalOnOff,
+  handleTextArea,
+  handleFileChange,
+}) => {
+  let ReviewImg = null;
+  if (uploadImg.file !== '') {
+    ReviewImg = (
+      <ImgView>
+        <img src={uploadImg.previewURL} alt="" />
+      </ImgView>
+    );
+  }
+  return (
+    <>
+      {reviewModal.reviewUpload && (
+        <Background onClick={modalOnOff}>
+          <ModalContainer onClick={e => e.stopPropagation()}>
+            <Title>리뷰 작성</Title>
+            <SelectionBar />
+            {ReviewImg}
+            <ReviewForm>
```

Comment:
```suggestion
}) => {

  return (
    <>
      {reviewModal.reviewUpload && (
        <Background onClick={modalOnOff}>
          <ModalContainer onClick={e => e.stopPropagation()}>
            <Title>리뷰 작성</Title>
            <SelectionBar />
            {uploadImg.file !== '' && <ImgView>
              <img src={uploadImg.previewURL} alt="" />
            </ImgView>}
            <ReviewForm>
```
저희 truthy 조건의 조건부 렌더링은 이렇게 하기로 하지 않았나요? 🤔

Reply:
> 저희 truthy 조건의 조건부 렌더링은 이렇게 하기로 하지 않았나요? 🤔

바로 수정했습니다.